core/Utils.py

```diff
@@ -94,3 +96,37 @@ def print_yaml_nicely(cls, to_print):
         """
         import json
         print json.dumps(to_print, indent=2)
+
+"""
+Used for re-typing variables
```

Comment:
Not sure if this should be placed in the neuron instead of Utils. Furthermore, those methods seems to prevent errors for french language only, right?